Adding adequate CLI examples for creating triggers (#6002)
influxdata/docs-v2: content/shared/influxdb3-cli/create/distinct_cache.md

| `INFLUXDB3_DATABASE_NAME` | `--database` |
| `INFLUXDB3_AUTH_TOKEN`    | `--token`    |

## Prerequisites

Before creating a distinct value cache, make sure you:

1. [Create a database](/influxdb3/version/reference/cli/influxdb3/create/database/)

2. [Create a table](/influxdb3/version/reference/cli/influxdb3/create/table/) that includes the columns you want to cache

3. Have a valid authentication token

## Examples

Before running the following commands, replace the placeholder values with your own:

- {{% code-placeholder-key %}}`DATABASE_NAME`{{% /code-placeholder-key %}}:

>   Database name

Reviewer: ```suggestion
  The database name
```

- {{% code-placeholder-key %}}`TABLE_NAME`{{% /code-placeholder-key %}}: 

>   Table name

Reviewer: ```suggestion
  The name of the table to cache values from
```

- {{% code-placeholder-key %}}`CACHE_NAME`{{% /code-placeholder-key %}}: 

>   Name of the distinct value cache to create

Reviewer: ```suggestion
  The name of the distinct value cache to create
```

> - {{% code-placeholder-key %}}`COLUMN_NAME`{{% /code-placeholder-key %}}: Column to 

Reviewer: ```suggestion
- {{% code-placeholder-key %}}`COLUMN_NAME`{{% /code-placeholder-key %}}: The column to 
```

cache distinct values from

You can also set environment variables (such as `INFLUXDB3_AUTH_TOKEN`) instead of passing options inline.

{{% code-placeholders "(DATABASE|TABLE|COLUMN|CACHE)_NAME" %}}

### Create a distinct cache for one column

Track unique values from a single column. This setup is useful for testing or simple use cases.

<!--pytest.mark.skip-->

```bash
influxdb3 create distinct_cache \
  --database DATABASE_NAME \
  --table TABLE_NAME \
  --column COLUMN_NAME \
  CACHE_NAME
```

### Create a hierarchical cache with constraints

> Create a distinct value cache for multiple columns. This configuration tracks unique combinations of `room` and `sensor_id`, and sets limits on the number of entries and their maximum age.

Reviewer: ```suggestion
Create a distinct value cache for multiple columns. The following example tracks unique combinations of `room` and `sensor_id`, and sets limits on the number of entries and their maximum age.
```
